src/docs/content/builders/hover-card.md

```diff
@@ -1,7 +1,6 @@
 ---
 title: Hover Card
-description:
-  Displays additional information or options when the cursor hovers over a particular element.
+description: Enable sighted users to preview content behind a link.
```

Comment:
```suggestion
description: Enable sighted users to preview content behind a link with mouse hover or keyboard focus.
```

Otherwise it's still not clear that touch events don't change it

Reply:
Would it be worth having a call out? Make it abundantly clear that this will not render for pointer event on mobile. It is purely for keyboard/mouse events

Reply:
@geerew Yeah I think that's a good idea tbh, to avoid any future issues around it. I'll add one somewhere toward the beginning of the docs!